pkgs/top-level/all-packages.nix

```diff
@@ -3678,9 +3678,8 @@ with pkgs;
 
   cudd = callPackage ../development/libraries/cudd { };
 
-  cue = callPackage ../development/tools/cue { };
-
-  writeCueValidator = callPackage ../development/tools/cue/validator.nix { };
+  cue = cuelang;
```

Comment:
I don't think it's necessary to rename it to `cuelang`, few distros have that as an attribute name. https://repology.org/project/cuelang/versions

Reply:
I was trying to package Cue, a music player: https://github.com/ravachol/cue
Here is it: https://github.com/NixOS/nixpkgs/pull/249095

That being said, I think it can be useful to differentiate them even being rarely used.

Reply:
`cue` has been renamed to `kew`. See https://github.com/ravachol/kew/commit/2334207e83f68d9fd781997be5db4c410ef4f865.

Reply:
OK, "reverting" the rename.